resources/definitions/tinyboy_e10.def.json

```diff
@@ -0,0 +1,32 @@
+{
+    "version": 2,
+    "name": "TinyBoy E10_J10_L10_M10",
+    "inherits": "fdmprinter",
+    "metadata": { 
+		"visible": true,
+        "author": "Fred Chan",
+        "manufacturer": "TinyBoy / mago",
+        "category": "tinyboy",		
```

Comment:
Please don't write down a category. This is no longer used. I'll add this to our tests to prevent future confusion.

Reply:
updated, thanks

Reply:
only change a little lines but can't get compile from github, seem my source is not update from master, how can I get the master again from github cura ? Thanks

Reply:
See instructions for updating a fork of a repository here: https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork

Reply:
The current issue with the failed tests (the X) is not really your problem. It seems that you forked off at a time when we accidentally had failing tests ourselves. You can ignore that for now.

Reply:
thanks a lot